_search-plugins/star-tree-index.md

```diff
@@ -0,0 +1,189 @@
+
+```json
+PUT logs
+{
+  "settings": {
+    "index.number_of_shards": 1,
+    "index.number_of_replicas": 0,
+    "index.composite_index": true
+  },
+  "mappings": {
+    "composite": {
+      "request_aggs": {
+        "type": "star_tree",
+        "config": {
+          "ordered_dimensions": [
+            {
+              "name": "status"
+            },
+            {
+              "name": "port"
+            }
+          ],
+          "metrics": [
+            {
+              "name": "size",
+              "stats": [
+                "sum"
+              ]
+            },
+            {
+              "name": "latency",
+              "stats": [
+                "avg"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "properties": {
+      "status": {
+        "type": "integer"
+      },
+      "port": {
+        "type": "integer"
+      },
+      "size": {
+        "type": "integer"
+      },
+      "latency": {
+        "type": "scaled_float",
+        "scaling_factor": 10
+      }
+    }
+  }
+}
+```
+
+For detailed information about star-tree index mapping and parameters see [star-tree field type]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/star-tree/).
+
+## Supported query and aggregations
+
+Star-tree indexes can be used to optimize queries and aggregations. 
+
+### Supported queries
+The following queries are supported as of OpenSearch 2.18:
+
+- [Term query](https://opensearch.org/docs/latest/query-dsl/term/term/)
+- [Match all docs query](https://opensearch.org/docs/latest/query-dsl/match-all/)
+
+To use queries, the queries fields must present in the `ordered_dimensions` section of the star-tree configuration.
+
+### Supported aggregations
+ 
+Following metric aggregations are supported as of OpenSearch 2.18.
+- [Sum](https://opensearch.org/docs/latest/aggregations/metric/sum/)
+- [Minimum](https://opensearch.org/docs/latest/aggregations/metric/minimum/)
+- [Maximum](https://opensearch.org/docs/latest/aggregations/metric/maximum/)
+- [Value count](https://opensearch.org/docs/latest/aggregations/metric/value-count/)
+- [Average](https://opensearch.org/docs/latest/aggregations/metric/average/)
+
+To use aggregations:
+
+- The fields must present in the `metrics` section of the star-tree configuration.
+- The metric aggregation type must be part of the `stats` parameter.
+
+### Aggregation example
+
+The following example gets the sum of the `size` field for all error logs with `status=500`, using the [example mapping](#example-mapping):
+
+```json
+POST /logs/_search
+{
+  "query": {
+    "term": {
+      "status": "500"
+    }
+  },
+  "aggs": {
+    "sum_size": {
+      "sum": {
+        "field": "size"
+      }
+    }
+  }
+}
+```
+
+With the star-tree index, the result will be retrieved from a single aggregated document as it traverses to the `status=500` node, as opposed to scanning through all the matching documents. This will results in lower query latency.
```

Comment:
Should "to the" be included here, or should it just be "as it traverses the `status=500` node"? In the second sentence, either "This will result" or "This results".